### ISSUE TITLE
How to adaptive ratio is learned by indices function?

### ISSUE THREAD

**Issue by flandrewries** [oucailab/AFENet#5]
We noticed the rh and rw are utilized as indices for constructing the the separation mask for high-frequency and low-frequency feature map.
However, in your code, the mask is manually assigned for value one and zero, based on different ranges.
This operation do not back-propagated by gradient-descending loss functions.
Therefore, your rw and rh maybe always an constant for all training steps.
So, the Adaptive Frequency Separation is actually a fixed separation with the initialized 0.5 ranges.
